pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -286,27 +312,22 @@ def connection_test(context):
 
 
 @connection_group.command('save', options_metavar=CMD_OPTS_TXT)
-@click.option('-n', '--name', type=str, metavar="NAME",
-              default=None,
-              help='Name for the new WBEM connection definition.')
+@click.argument('name', type=str, metavar='NAME', required=True)
 @add_options(verify_option)
 @click.pass_obj
-def connection_save(context, **options):
+def connection_save(context, name, **options):
     """
     Save the current connection as a new WBEM connection definition.
 
-    Create a new WBEM connection definition in the connections file from
-    the current connection. A connection definition with that name must not
-    yet exist.
-
-    This command is useful when you have defined a connection on the command
-    line and want to save it in the connections file for later use.
+    Create a new WBEM connection definition in the connections file from the
+    current connection with the NAME argument as connection name. A connection
```

Comment:
On " with the NAME argument as connection name":
1. To be precise, it is the connection definition name.
2. Along with my suggestion on "connection add", I suggest to shorten the first sentence to: "Create a new WBEM connection definition named NAME in the connections file from the current connection."

Reply:
I shortened your proposal by leaving off the "in the connections file from the current connection. Note that you proposed above to put the --name option in here also to get from existing so it was more than just current.  However, the real reason is that we have a real limit on that first sentence so it shows up as a complete sentence on a single line in the list of Commands:

Reply:
DONE